README.md

```diff
@@ -168,305 +116,247 @@ Typical Java threads
 
-##### IO
+* #### IO
 
-The IO stage is when data is available for output. This is a blocking point in the workflow as
+  * The IO stage is when data is available for output. This is a blocking point in the workflow as
 all items in the payload family tree must be prepared to transition to the IO stage
 before we can proceed.
 
-##### Review
+* #### Review
 
-Post IO most of the currentForms are stripped off as they are handled by the IO places.
+  * Post IO most of the currentForms are stripped off as they are handled by the IO places.
 The itinerary is available and could be used in post-processing provenance events.
 
-#### Routing
+### Routing
 
 Decisions on how data gets sent to Places revolves around the concept of DirectoryEntry, which is also referred to as a 
 Key within the code.  A DirectoryEntry is of the form:
-
 FORM.SERVICENAME.STAGE.URL$EXPENSE
 
 Routing uses a combination of the FORM, STAGE and EXPENSE to decide where to send the data.  The STAGEs define an order 
 for the processing flow within the framework.  Places are bound to specific STAGEs and will not be run outside of that 
 STAGE.  The current STAGE list and order is:
 
-Actual name | Description | Run in parallel?
-------------|-------------|------------------
-STUDY | prepare, coordinate idents | No
-ID | identification phase | No
-COORDINATE | Coordinate processing | No
-PRETRANSFORM | before transform hook | Yes
-TRANSFORM | transformation phase | No
-POSTTRANSFORM | after transform hook | Yes
-ANALYZE | analysis/metadata generation | Yes
-VERIFY | verify for output | No
-IO | output | No
-REVIEW |  finish off | No
-
-#### Parsers vs Places
-
-Emissary Places are for processing, transforming, identifying, analyzing data of interest. As items
+### Parsers vs Places
+
+Emissary Places are for identifying, processing, transforming, and analyzing data of interest. 
+
+As items
 are unwrapped and processed the results are all kept together in a "family tree". This structure
-is a List<IBaseDataObject> in the code. The list can be sorted in family order by using the
-emissary.util.ShortNameComparator. 
+is a List\<IBaseDataObject> in the code.
 
-Sometimes, for our own convenience, or for dealing with formats provided by up-stream systems,
-the data arrives packaged in various formats. These containers can have multiple items that
-should not be logically related in a family tree. The Emissary Parser framework deals with this
-situation.
+The Emissary Parser framework is designed to extract information from any data containers/wrappers on the payload that may contain information useful for identification or processing.
 
-Parsers need to make a quick identification of the format of the container. This is called
-out by the emissary.parser.ParserFactory and uses the emissary.parser.DataIdentifier as the
+Parsers need to make a quick identification of the container format (i.e. Apache NiFi). This is called
```

Comment:
Listed a practical example of a container format to help users better understand how this applies

Reply:
Apache NiFi FlowFile?